eng/pipelines/common/global-build-job.yml

```diff
@@ -25,19 +25,11 @@ jobs:
     variables: ${{ parameters.variables }}
 
     steps:
-    - ${{ if eq(parameters.osGroup, 'OSX') }}:
-      - script: |
-          $(setScriptToEchoAndFailOnNonZero) 
-          brew install pkgconfig icu4c openssl autoconf automake libtool pkg-config python3
-          brew link --force icu4c
-          ln -s /usr/local/opt/openssl/lib/pkgconfig/libcrypto.pc /usr/local/lib/pkgconfig/
-          ln -s /usr/local/opt/openssl/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
-          ln -s /usr/local/opt/openssl/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
-        displayName: Install Build Dependencies
-
     - template: /eng/pipelines/common/clone-checkout-bundle-step.yml
 
-    - ${{ if in(parameters.osGroup, 'OSX', 'iOS','tvOS') }}: 
+    - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
+      - script: sh $(Build.SourcesDirectory)/eng/install-native-dependencies.sh ${{ parameters.osGroup }}
+        displayName: Install Build Dependencies
```

Comment:
Nit, empty line for readability.

Reply:
Fixed in c6f19d622e7a713fdb1a2b84fc2ae45137704cb5.